nixos/modules/tasks/encrypted-devices.nix

```diff
@@ -65,12 +72,11 @@ in
     boot.initrd = {
       luks = {
         devices =
-          builtins.listToAttrs (map (dev: { name = dev.encrypted.label; value = { device = dev.encrypted.blkDev; }; }) keylessEncDevs);
+          builtins.listToAttrs (map (dev: with dev.encrypted; { name = label; value = { device = blkDev; }; }) keylessEncDevs);
```

Comment:
I actually prefer the longer variant because it makes clear where `label`, `blkDev` and `keylessEncDevs` are coming from.

Reply:
If you want to make it more readable, you can introduce a line break after `dev: {`

Reply:
Done.